CMakeLists.txt

```diff
@@ -1565,7 +1564,7 @@ if(HPX_WITH_APEX)
   include(GitExternal)
   git_external(apex
     https://github.com/khuck/xpress-apex.git
-    v2.0.1
+    develop   # this should be set to a known good tag when HPX is released!
```

Comment:
Should we turn this into a CMake configuration variable? May I ask you to at least add a note to the release procedure here: https://github.com/STEllAR-GROUP/hpx/blob/master/docs/release_procedure.rst?